Adding relative pagination and an icon

okofish opened the pull request:
With this PR you can move relative to the current page with /goto/next and /goto/prev.
I also added a stylish icon so it looks better in the dock :)

Apologies for any silly style mistakes; I don't actually know Objective-C.

okofish committed:
Adding a handsome-looking icon

okofish committed:
Adding relative pagination: /goto/next and /goto/prev

ChrisAshworth commented:
Hi Jessie, thanks for the pull request! Fun addition.

Two comments:
- The [self performSelectorOnMainThread:withObject:waitUntilDone:NO] methods can be simplified to just calling the method directly.  By the time we're in processOscMessage we're already on the main thread, so don't need to go there again.
- While I shan't disagree that the icon is handsome (thanks! :-) ), for branding purposes I think we'd want to create something that is more meaningfully about the function of the app rather than the generic logo for our company.

okofish committed:
Calling pdfView methods directly

okofish committed:
Removing icons (for the time being)

okofish commented:
Thanks for the response! As you'll see, I removed the icons and changed the method references. Objective-C is very different from the languages I regularly work with, and I'm not really familiar with the general syntax.

ChrisAshworth commented:
Thanks Jesse!
